Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,199 @@
+<!--
+Luis Daniel Rady Colunga
+
+Ejercicio #1 -> WebDevEx1
+Parte 3 de http://www.pairuptocode.com/exercises/htmlbasics.html
+Girls Developing It
+
+Fecha de realización: 09/05/18
+Última modificación: 10/05/18
+-->
+
+<!DOCTYPE html>
+<html lang="">
+    <head>
+          <meta charset="utf-8">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+          
+          <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+          <link rel="stylesheet" href="css/style.css">
+          <script src="js/index.js"></script>
+          
+          <title>GirlDevelopIt</title>
+    </head>
+    
+    <body>
+       <!--Inicio del header,
+           Incluye el logo y el separador en diagonal -->
+        <header>
+            <img src="img/girldevelopit_logo.png" class="logo" alt="logo">
+            <img src="img/divider-triangle-header.png" class="divider">
+        </header>
+        
+        
+        <!--Inicio del contenido,
+            Se encuentra dividido en 3 secciones, la sección de about,
+            la sección de lugares y la sección de próximos cursos más el registro -->
+        <content>
```

Comment:
Try to add a `container` class to all your content to have some standard margin in all your content
https://getbootstrap.com/docs/4.1/layout/overview/#containers